training/finetune_acft_hf_nlp/components/preprocess/question_answering/spec.yaml

```diff
@@ -0,0 +1,110 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+name: question_answering_datapreprocess
+version: 0.0.5
+type: command
+
+is_deterministic: True
+
+display_name: Question Answering DataPreProcess
+description: Component to preprocess data for question answering task
+
+environment: azureml:acft-hf-nlp-gpu:2
```

Comment:
Please use MCR asset URI for all the components.

Reply:
updated the environment asset URI